### PR TITLE
Offline analysis

### PR DESCRIPTION
**Pull request by christianp**
@Lycanic can you review this as well, please?

We'll also need some documentation.

### CODE REVIEW

File: editor/static/js/numbas/numbas-runtime.js
Comment (Lycanic) [inline editor/static/js/numbas/numbas-runtime.js:1]: Are these adjustments to the reading of expressions part of the results-download, or something done alongside?

**Review by Lycanic — APPROVED** [review]
Changes work and seem intuitive. I will add documentation but that can probably be a later pull?

File: editor/static/js/numbas/numbas-runtime.js
Comment (christianp) [reply editor/static/js/numbas/numbas-runtime.js:1]: eugh, that's meant to be on a different branch. Now merged into master, so ignore it.

**Comment by Lycanic** [comment]
I believe we still need a link to analysis.html from the editor page for the question. Perhaps in organisation?

**Pull request merged**